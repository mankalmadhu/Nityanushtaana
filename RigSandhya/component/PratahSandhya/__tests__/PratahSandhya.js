import React from 'react';
import ಪ್ರಾತಸ್ಸ೦ಧ್ಯಾ from '../PratahSandhya';
import Renderer from 'react-test-renderer';

jest.mock('../../Lekhana/Lekhana', () => 'ಲೇಖನ' );
jest.mock('../../Puta/Puta', () => 'ಪುಟ' );
jest.mock('../../Ghatta/Sochane/AasanaSoochane', () => 'ಆಸನಸೂಚನೆ' );
jest.mock('../../Ghatta/Aachamaneya/Aachamaneya', () => 'ಆಚಮನೀಯ' );
jest.mock('../../Ghatta/Prokshana/Prokshana', () => 'ಪ್ರೋಕ್ಷಣ' );
jest.mock('../../Ghatta/Sankalpa/PratahSandhyaSankalpa', () => 'ಪ್ರಾತಃಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ' );
jest.mock('../../Ghatta/Marjane/PratahMarjane', () => 'ಪ್ರಾತಃಮಾರ್ಜನೆ' );
jest.mock('../../Ghatta/DwiteyaMarjane/DwiteyaMarjane', () => 'ದ್ವಿತೀಯಮಾರ್ಜನೆ' );
jest.mock('../../Ghatta/DwiteyaMarjane/Visarjane', () => 'ವಿಸರ್ಜನೆ' );

test("ಪ್ರಾತಸ್ಸ೦ಧ್ಯಾ", () => {
  expect(Renderer.create(
    <ಪ್ರಾತಸ್ಸ೦ಧ್ಯಾ />
  ).toJSON()).toMatchSnapshot();

});