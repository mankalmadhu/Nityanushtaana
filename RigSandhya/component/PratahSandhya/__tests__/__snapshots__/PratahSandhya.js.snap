// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪ್ರಾತಸ್ಸ೦ಧ್ಯಾ 1`] = `
<ಲೇಖನ>
  <ಪುಟ>
    <ಆಸನಸೂಚನೆ />
    <ಆಚಮನೀಯ />
  </ಪುಟ>
  <ಪುಟ>
    <ಪ್ರೋಕ್ಷಣ />
    <ಆಚಮನೀಯ />
  </ಪುಟ>
  <ಪುಟ>
    <ಪ್ರಾತಃಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ />
  </ಪುಟ>
  <ಪುಟ>
    <ಪ್ರಾತಃಮಾರ್ಜನೆ />
  </ಪುಟ>
  <ಪುಟ>
    <ಆಚಮನೀಯ />
  </ಪುಟ>
  <ಪುಟ>
    <ದ್ವಿತೀಯಮಾರ್ಜನೆ />
    <ವಿಸರ್ಜನೆ />
  </ಪುಟ>
  <ಪುಟ>
    <ಆಚಮನೀಯ />
  </ಪುಟ>
  <ಪುಟ>
    <ಪ್ರಾತರ್ರಘ್ಯಸ೦ಕಲ್ಪ />
  </ಪುಟ>
  <ಪುಟ>
    <ಪ್ರಾತರರ್ಘ್ಯ />
  </ಪುಟ>
  <ಪುಟ>
    <ಪ್ರಾತಃಗಾಯತ್ರೀ />
  </ಪುಟ>
  <ಪುಟ>
    <ಪ್ರಾತರುಪಸ್ಥಾನ />
  </ಪುಟ>
</ಲೇಖನ>
`;
